media-plugins/swh-lv2/swh-lv2-1.0.16.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Large collection of LV2 audio plugins/effects"
+HOMEPAGE="http://plugin.org.uk/"
+SRC_URI="https://github.com/swh/lv2/archive/v${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 x86"
+IUSE=""
+
+RDEPEND=">=sci-libs/fftw-3.0.0"
```

Comment:
The old dep wasn't pretty but this one is worse I think

Please use: sci-libs/fftw:3.0